Subject: [PATCH] [FEAT VS1804] README.md: remove useless empty line
From: Marc Hartmayer <mhartmay@linux.ibm.com>

Summary:     genprotimg: Introduce new tool for the creation of PV images
Description: genprotimg takes a kernel, host-key documents, optionally an
             initrd, optionally a file with the kernel command line, and it
             generates a single, loadable image file. The image consists of a
             concatenation of a plain text boot loader, the encrypted
             components for kernel, initrd, and cmdline, and the
             integrity-protected PV header, containing metadata necessary for
             running the guest in PV mode. It's possible to use this image file
             as a kernel for zIPL or for a direct kernel boot using QEMU.
Upstream-ID: b06af6026f08d67339a109ba7457373ab82d3248
Problem-ID:  VS1804

Upstream-Description:

             README.md: remove useless empty line

             Remove useless empty line.

             Reviewed-by: Jan Höppner <hoeppner@linux.ibm.com>
             Signed-off-by: Marc Hartmayer <mhartmay@linux.ibm.com>
             Signed-off-by: Jan Höppner <hoeppner@linux.ibm.com>


Signed-off-by: Marc Hartmayer <mhartmay@linux.ibm.com>
---
 README.md |    1 -
 1 file changed, 1 deletion(-)

--- a/README.md
+++ b/README.md
@@ -1,4 +1,3 @@
-
 s390-tools
 ==========
 
